components/textbox/textbox.jsx

```diff
@@ -77,6 +77,12 @@ export default class Textbox extends React.Component {
         this.props.onChange(e);
     }
 
+    componentDidUpdate() {
+        if (this.props.preview) {
+            this.refs.preview.focus();
```

Comment:
I'm worried this might steal the focus from something else if unrelated props happen to trigger an update, e.g. channel switching while previewing a post in the RHS.

Reply:
@jespino The preview mode for the RHS is handled separately right?
And yes, it is intentional that we want to exist the preview mode right away after sending the message.

Reply:
Sorry, I don't follow your response -- the code above will force the preview to take the focus whenever the component re-renders, since we're not gating the change on an actual props update. I believe this has the risk of stealing focus unexpectedly if the component happens to re-render while previewing. Or perhaps I'm misunderstanding?